webrender/res/ps_border_edge.vs.glsl

```diff
@@ -42,11 +42,15 @@ void write_color(vec4 color, float style, bool flip) {
 
     switch (int(style)) {
         case BORDER_STYLE_GROOVE:
+        {
```

Comment:
Forget to explain this case. See [1], ANGLE will translate ternary operator to ```type s; if(b) s = x; else s = y;```. And ```type s``` is a declaration.

[1]: https://github.com/google/angle/blob/d7b1ab5893953b97fe5dc1b3087a9b4b50ec45ee/src/compiler/translator/UnfoldShortCircuitToIf.cpp#L139